test/style-camel-case.js

```diff
@@ -0,0 +1,17 @@
+var test = require("tape")
+var h = require("../h.js")
+var diff = require("../diff.js")
+var patch = require("../patch.js")
+var createElement = require("../create-element.js")
+
+test("convert camel-case properties to dashes in string form", function (assert) {
```

Comment:
This is a good test :)

We should fix `.outerHTML` and `.toString()` in `min-document`.